src/transform/split-declaraion.js

```diff
@@ -0,0 +1,28 @@
+import traverser from '../traverser';
+import multiReplaceStatement from '../utils/multi-replace-statement';
+import VariableDeclaration from '../syntax/variable-declaration';
+
+export default function(ast) {
+  traverser.replace(ast, {
```

Comment:
sorry, i just take the code of `let` as reference. Therefore, it seems the problem exists somewhere else